lib/signs_ui_web/controllers/signs_controller.ex

```diff
@@ -7,17 +7,28 @@ defmodule SignsUiWeb.SignsController do
   end
 
   def update(conn, params) do
-    updated_signs = params
-                    |> get_enabled_map()
-                    |> SignsUI.Signs.State.update()
-    conn
-    |> put_flash(:success, "Signs updated successfully")
-    |> render("index.html", signs: updated_signs)
+    conn = perform_update(conn, params)
+    redirect(conn, to: "/")
```

Comment:
The usual flow is redirect on success, render on failure. The advantage of that approach is that if it fails for some reason (maybe a blip in S3), the app state won't change, but the _form_ at least will keep the changes they made. If the error is something like "please try again", then all they need to do is click submit again.

Reply:
Yeah, I wasn't sure if it'd be confusing if it looked the changes went through. But it would be annoying to reset all the checkboxes. Hopefully the message should make it clear